Could not find a declaration file for module '@gltf-transform/core' [Node.js/Typescript]

**Describe the bug**
@gltf-transform/core and @gltf-transform/extension cannot be imported when using `"module": "NodeNext"` and `"target": "ESNext"` in `tsconfig.json`. You can find the compiler output in the next section. This setup does not work with version 3.0.0 but builds with version 2.5.1 (requires to also install `draco3dgltf` and its types first).

**To Reproduce**
I set up a minimal project you can find [here](https://github.com/aleneum/gltf-test)

#### package.json
```json
{
    "type": "module",
    "scripts": {
        "build": "tsc"
    },
    "dependencies": {
        "@gltf-transform/core": "^3.0.0",
        "@gltf-transform/extensions": "^3.0.0"
    },
    "devDependencies": {
        "typescript": "^4.9.5"
    }
}
```
#### tsconfig.json
```json
{
  "compilerOptions": {
    "target": "ESNext",
    "module": "NodeNext",
    "esModuleInterop": true,
    "strict": true,
    "outDir": "build"
  }
}
```
#### src/main.ts
```typescript
import { Document, NodeIO, WebIO } from "@gltf-transform/core";
import { KHRONOS_EXTENSIONS } from "@gltf-transform/extensions";
```

#### Steps to reproduce
```shell
git clone git@github.com:aleneum/gltf-test.git
cd gltf-test
npm install
npm run  build
```
#### Output
```
src/main.ts:1:41 - error TS7016: Could not find a declaration file for module '@gltf-transform/core'. '/Users/aneumann9/workspace/gltf-test/node_modules/@gltf-transform/core/dist/core.modern.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/gltf-transform__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@gltf-transform/core';`

1 import { Document, NodeIO, WebIO } from "@gltf-transform/core";
                                          ~~~~~~~~~~~~~~~~~~~~~~

src/main.ts:2:36 - error TS7016: Could not find a declaration file for module '@gltf-transform/extensions'. '/Users/aneumann9/workspace/gltf-test/node_modules/@gltf-transform/extensions/dist/extensions.modern.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/gltf-transform__extensions` if it exists or add a new declaration (.d.ts) file containing `declare module '@gltf-transform/extensions';`

2 import { KHRONOS_EXTENSIONS } from "@gltf-transform/extensions";
                                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 2 errors in the same file, starting at: src/main.ts:1
```

**Expected behavior**
Import should work

**Versions:**
 - Version: 3.0.0
 - Environment: Node.js (v19.6.0), Typescript (4.9.5)

--- a/docs/CLI_HELP.md
+++ b/docs/CLI_HELP.md
@@ -1,5 +1,5 @@
 
-  gltf-transform 3.0.0 — Command-line interface (CLI) for the glTF Transform SDK.
+  gltf-transform 3.0.1-alpha.0 — Command-line interface (CLI) for the glTF Transform SDK.
 
   USAGE 
   

--- a/lerna.json
+++ b/lerna.json
@@ -1,5 +1,5 @@
 {
-  "version": "3.0.0",
+  "version": "3.0.1-alpha.0",
   "npmClient": "yarn",
   "useWorkspaces": true,
   "packages": [

--- a/packages/cli/package.json
+++ b/packages/cli/package.json
@@ -1,6 +1,6 @@
 {
 	"name": "@gltf-transform/cli",
-	"version": "3.0.0",
+	"version": "3.0.1-alpha.0",
 	"repository": "github:donmccurdy/glTF-Transform",
 	"homepage": "https://gltf-transform.donmccurdy.com/cli.html",
 	"description": "CLI interface to glTF Transform",
@@ -26,9 +26,9 @@
 	},
 	"dependencies": {
 		"@caporal/core": "^2.0.2",
-		"@gltf-transform/core": "^3.0.0",
-		"@gltf-transform/extensions": "^3.0.0",
-		"@gltf-transform/functions": "^3.0.0",
+		"@gltf-transform/core": "^3.0.1-alpha.0",
+		"@gltf-transform/extensions": "^3.0.1-alpha.0",
+		"@gltf-transform/functions": "^3.0.1-alpha.0",
 		"@types/inquirer": "^9.0.3",
 		"@types/language-tags": "^1.0.1",
 		"@types/micromatch": "^4.0.2",

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -1,51 +1,55 @@
 {
-    "name": "@gltf-transform/core",
-    "version": "3.0.0",
-    "repository": "github:donmccurdy/glTF-Transform",
-    "homepage": "https://gltf-transform.donmccurdy.com/",
-    "description": "glTF 2.0 SDK for JavaScript and TypeScript, on Web and Node.js.",
-    "author": "Don McCurdy <dm@donmccurdy.com>",
-    "license": "MIT",
-    "type": "module",
-    "sideEffects": false,
-    "main": "./dist/core.cjs",
-    "module": "./dist/core.modern.js",
-    "exports": "./dist/core.modern.js",
-    "source": "./src/core.ts",
-    "types": "./dist/core.d.ts",
-    "browserslist": [
-        "defaults",
-        "not IE 11",
-        "node >= 14"
-    ],
-    "scripts": {
-        "dist": "microbundle --format modern,cjs --define PACKAGE_VERSION=$npm_package_version",
-        "watch": "microbundle watch --format modern,cjs --define PACKAGE_VERSION=$npm_package_version",
-        "watch:debug": "microbundle watch --format modern,cjs --define PACKAGE_VERSION=$npm_package_version --no-compress"
-    },
-    "keywords": [
-        "gltf",
-        "3d",
-        "model",
-        "webgl",
-        "threejs"
-    ],
-    "files": [
-        "dist/",
-        "src/",
-        "README.md",
-        "package.json",
-        "package-lock.json"
-    ],
-    "browser": {
-        "fs": false,
-        "path": false
-    },
-    "dependencies": {
-        "property-graph": "^1.0.0"
-    },
-    "mangle": {
-        "regex": "^_"
-    },
-    "gitHead": "917f61e8806556055aefc7fdf6e636c56ecb62d6"
+	"name": "@gltf-transform/core",
+	"version": "3.0.0",
+	"repository": "github:donmccurdy/glTF-Transform",
+	"homepage": "https://gltf-transform.donmccurdy.com/",
+	"description": "glTF 2.0 SDK for JavaScript and TypeScript, on Web and Node.js.",
+	"author": "Don McCurdy <dm@donmccurdy.com>",
+	"license": "MIT",
+	"type": "module",
+	"sideEffects": false,
+	"exports": {
+		"types": "./dist/core.d.ts",
+		"require": "./dist/core.cjs",
+		"default": "./dist/core.modern.js"
+	},
+	"types": "./dist/core.d.ts",
+	"main": "./dist/core.cjs",
+	"module": "./dist/core.modern.js",
+	"source": "./src/core.ts",
+	"browserslist": [
+		"defaults",
+		"not IE 11",
+		"node >= 14"
+	],
+	"scripts": {
+		"dist": "microbundle --format modern,cjs --define PACKAGE_VERSION=$npm_package_version",
+		"watch": "microbundle watch --format modern,cjs --define PACKAGE_VERSION=$npm_package_version",
+		"watch:debug": "microbundle watch --format modern,cjs --define PACKAGE_VERSION=$npm_package_version --no-compress"
+	},
+	"keywords": [
+		"gltf",
+		"3d",
+		"model",
+		"webgl",
+		"threejs"
+	],
+	"files": [
+		"dist/",
+		"src/",
+		"README.md",
+		"package.json",
+		"package-lock.json"
+	],
+	"browser": {
+			"fs": false,
+			"path": false
+	},
+	"dependencies": {
+		"property-graph": "^1.0.0"
+	},
+	"mangle": {
+		"regex": "^_"
+	},
+	"gitHead": "917f61e8806556055aefc7fdf6e636c56ecb62d6"
 }

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -1,6 +1,6 @@
 {
 	"name": "@gltf-transform/core",
-	"version": "3.0.0",
+	"version": "3.0.1-alpha.0",
 	"repository": "github:donmccurdy/glTF-Transform",
 	"homepage": "https://gltf-transform.donmccurdy.com/",
 	"description": "glTF 2.0 SDK for JavaScript and TypeScript, on Web and Node.js.",
@@ -42,8 +42,8 @@
 		"package-lock.json"
 	],
 	"browser": {
-			"fs": false,
-			"path": false
+		"fs": false,
+		"path": false
 	},
 	"dependencies": {
 		"property-graph": "^1.0.0"

--- a/packages/extensions/package.json
+++ b/packages/extensions/package.json
@@ -8,11 +8,15 @@
 	"license": "MIT",
 	"type": "module",
 	"sideEffects": false,
+	"exports": {
+		"types": "./dist/extensions.d.ts",
+		"require": "./dist/extensions.cjs",
+		"default": "./dist/extensions.modern.js"
+	},
+	"types": "./dist/extensions.d.ts",
 	"main": "./dist/extensions.cjs",
 	"module": "./dist/extensions.modern.js",
-	"exports": "./dist/extensions.modern.js",
 	"source": "./src/extensions.ts",
-	"types": "./dist/extensions.d.ts",
 	"browserslist": [
 		"defaults",
 		"not IE 11",

--- a/packages/extensions/package.json
+++ b/packages/extensions/package.json
@@ -1,6 +1,6 @@
 {
 	"name": "@gltf-transform/extensions",
-	"version": "3.0.0",
+	"version": "3.0.1-alpha.0",
 	"repository": "github:donmccurdy/glTF-Transform",
 	"homepage": "https://gltf-transform.donmccurdy.com/extensions.html",
 	"description": "Adds extension support to @gltf-transform/core",
@@ -35,7 +35,7 @@
 		"threejs"
 	],
 	"dependencies": {
-		"@gltf-transform/core": "^3.0.0",
+		"@gltf-transform/core": "^3.0.1-alpha.0",
 		"ktx-parse": "^0.4.5"
 	},
 	"files": [

--- a/packages/functions/package.json
+++ b/packages/functions/package.json
@@ -1,6 +1,6 @@
 {
 	"name": "@gltf-transform/functions",
-	"version": "3.0.0",
+	"version": "3.0.1-alpha.0",
 	"repository": "github:donmccurdy/glTF-Transform",
 	"homepage": "https://gltf-transform.donmccurdy.com/functions.html",
 	"description": "Functions for common glTF modifications, written using the core API",
@@ -35,8 +35,8 @@
 		"threejs"
 	],
 	"dependencies": {
-		"@gltf-transform/core": "^3.0.0",
-		"@gltf-transform/extensions": "^3.0.0",
+		"@gltf-transform/core": "^3.0.1-alpha.0",
+		"@gltf-transform/extensions": "^3.0.1-alpha.0",
 		"ktx-parse": "^0.4.5",
 		"ndarray": "^1.0.19",
 		"ndarray-lanczos": "^0.2.0",

--- a/packages/functions/package.json
+++ b/packages/functions/package.json
@@ -8,11 +8,15 @@
 	"license": "MIT",
 	"type": "module",
 	"sideEffects": false,
+	"exports": {
+		"types": "./dist/functions.d.ts",
+		"require": "./dist/functions.cjs",
+		"default": "./dist/functions.modern.js"
+	},
+	"types": "./dist/index.d.ts",
 	"main": "./dist/functions.cjs",
 	"module": "./dist/functions.modern.js",
-	"exports": "./dist/functions.modern.js",
 	"source": "./src/index.ts",
-	"types": "./dist/index.d.ts",
 	"browserslist": [
 		"defaults",
 		"not IE 11",

--- a/packages/test-utils/package.json
+++ b/packages/test-utils/package.json
@@ -1,7 +1,7 @@
 {
 	"private": true,
 	"name": "@gltf-transform/test-utils",
-	"version": "3.0.0",
+	"version": "3.0.1-alpha.0",
 	"type": "module",
 	"sideEffects": false,
 	"exports": "./dist/test-utils.modern.js",
@@ -12,8 +12,8 @@
 		"watch": "microbundle watch --format modern --no-compress"
 	},
 	"dependencies": {
-		"@gltf-transform/core": "^3.0.0",
-		"@gltf-transform/extensions": "^3.0.0",
-		"@gltf-transform/functions": "^3.0.0"
+		"@gltf-transform/core": "^3.0.1-alpha.0",
+		"@gltf-transform/extensions": "^3.0.1-alpha.0",
+		"@gltf-transform/functions": "^3.0.1-alpha.0"
 	}
 }

